wallet/createtx.go

```diff
@@ -369,7 +369,7 @@ func (w *Wallet) txToOutputsInternal(op errors.Op, outputs []*wire.TxOut, accoun
 		}
 		var err error
 		atx, err = txauthor.NewUnsignedTransaction(outputs, txFee,
-			inputSource.SelectInputs, changeSource)
+			inputSource.SelectInputs, changeSource, w.ChainParams().MaxTxSize)
```

Comment:
you can use the unexported .chainParams field instead since this is in the wallet package.